pandas/io/formats/format.py

```diff
@@ -370,6 +370,15 @@ def _get_formatter(self, i):
                 i = self.columns[i]
             return self.formatters.get(i, None)
 
+    def _format_col(self, i):
+        frame = self.tr_frame
```

Comment:
can you add a doc-string here (types annotations also appreciated! but just do if you can)